src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js

```diff
@@ -44,18 +47,14 @@ const defaultProps = {
 function BaseReportActionContextMenu(props) {
     const [shouldKeepOpen, setShouldKeepOpen] = useState(false);
     const wrapperStyle = getReportActionContextMenuStyles(props.isMini, props.isSmallScreenWidth);
+
+    const reportAction = useMemo(() => {
+        if (_.isEmpty(props.reportActions) || props.reportActionID === '0') return {};
+        return props.reportActions[props.reportActionID];
```

Comment:
```suggestion
        return props.reportActions[props.reportActionID] || {};
```
It would be theoretically possible that the report action doesn't exist, right?